apps/ocean-api/src/modules/PlaygroundModule.ts

```diff
@@ -0,0 +1,65 @@
+import { Injectable, Logger, Module } from '@nestjs/common'
+import { ConfigService } from '@nestjs/config'
+import { Interval, ScheduleModule } from '@nestjs/schedule'
+import { BotLogger, Playground } from '@defichain/playground'
+import { ApiClient } from '@defichain/jellyfish-api-core'
+
+/**
+ * Provide a PlaygroundRunner when PLAYGROUND_ENABLE is enabled.
+ */
+@Module({
+  imports: [
+    ScheduleModule.forRoot()
+  ],
+  providers: [
+    {
+      provide: 'PLAYGROUND_RUNNER',
+      useFactory: (configService: ConfigService, apiClient: ApiClient): PlaygroundRunner | undefined => {
+        if (configService.get<boolean>('PLAYGROUND_ENABLE') === false) {
+          return undefined
+        }
+
+        if (configService.get<string>('API_NETWORK') !== 'regtest') {
```

Comment:
Not sure if any chance we can use this for testnet, like pre deployment test.

Reply:
Probably not since we need the privateKeys